engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -103,6 +103,7 @@
 {
   public static final String DT_VERSION = StreamingApplication.DT_PREFIX + "version";
   public static final String DT_DFS_ROOT_DIR = StreamingApplication.DT_PREFIX + "dfsRootDirectory";
+  public static final String APEX_DFS_ROOT_DIR = StreamingApplication.APEX_PREFIX + "dfsRootDirectory";
```

Comment:
Looks like this is specific to the application runtime. Can it be called APEX_APP_DFS_ROOT_DIR and be set to apex.app.dfsRootDir